nixos/modules/services/hardware/pcscd.nix

```diff
@@ -60,7 +54,7 @@ in
 
     systemd.services.pcscd = {
       environment.PCSCLITE_HP_DROPDIR = pluginEnv;
-      restartTriggers = [ "/etc/reader.conf" ];
+      restartTriggers = [ cfgFile ];
```

Comment:
Since `cfgFile` is used in `serviceConfig.ExecStart`, there is no need to add it to `restartTriggers`.

Reply:
```suggestion
```